.github/workflows/ci.yml

```diff
@@ -7,24 +7,20 @@ on:
 
 jobs:
   build_and_test:
+
     runs-on: ubuntu-latest
+
     strategy:
       matrix:
         node-version: [18.x]
-    name: Node ${{ matrix.node }}
 
     steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Setup node ${{ matrix.node-version }}
-        uses: actions/setup-node@v4
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
         with:
-          cache: 'npm'
           node-version: ${{ matrix.node-version }}
-      - name: install
-        run: npm ci
-      - name: build
-        run: npm run nx -- affected:build --base=origin/main --head=HEAD
-      - name: test
-        run: npm run nx -- affected:test --parallel=1 --base=origin/main --head=HEAD
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build-all
+      - run: npm run test-all
```

Comment:
This is momentary, We need to revert and use the nx affected for pull request but change the release pipeline to run a everything regardless of cache or affected.